MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -3297,7 +3298,7 @@ public string GetHwDeinterlaceFilter(EncodingJobInfo state, EncodingOptions opti
             return string.Empty;
         }
 
-        public string GetHwTonemapFilter(EncodingOptions options, string hwTonemapSuffix, string videoFormat)
+        private string GetHwTonemapFilter(EncodingOptions options, string hwTonemapSuffix, string videoFormat, bool isMjpegEncoder)
```

Comment:
I think `forceFullRange` or `useFullRange` is better because we might use this in other cases in the future.